syntax = "proto3";
package opencv_object_tracking;

message ClientInfo {
    string info = 1;
}

message ServerInfo {
    string info = 1;
}

message Location {
    int32 id = 1;
    int32 x = 2;
    int32 y = 3;
    int32 width = 4;
    int32 height = 5;
    int32 middle_inc = 6;
}

service LocationService {

    rpc registerClient (ClientInfo) returns (ServerInfo) {
    }

    rpc getLocations (ClientInfo) returns (stream Location) {
    }
}

